packages/auth/src/core/auth/auth_impl.ts

```diff
@@ -349,7 +351,23 @@ export class AuthImpl implements AuthInternal, _FirebaseService {
   }
 
   setRecaptchaConfig(config: RecaptchaConfig): void {
-    this._recaptchaConfig = config;
+    if (this.tenantId == null) {
+      this._agentRecaptchaConfig = config;
+    } else {
+      this._tenantRecaptchaConfigs[this.tenantId] = config;
+    }
+    if (config.emailPasswordEnabled === true) {
+      const verifier = new RecaptchaEnterpriseVerifier(this);
+      void verifier.verify();
```

Comment:
Is it necessary to void out the return value here for the linter? (I can't remember if that's one of the lint checks we have setup)

Reply:
Yeah, otherwise it will complain not handling the response.